src/Components/Referrer/Referrer.tsx

```diff
@@ -26,6 +27,16 @@ export const BrandedFooter = () => {
   }
   return <Footer />;
 };
+export const ResultsMessageForNeeds = () => {  
+  const { whiteLabel } = useParams();  
+  if (whiteLabel === 'nc')
```

Comment:
Instead of hard coding the white label, can we add a feature flag? Then you can add the feature flag to the config.
```suggestion
  if (getReferrer('featureFlags').includes('nc_show_211_link'))
```